cypress/integration/2.1.2-signed_in_user.spec.js

```diff
@@ -25,7 +26,7 @@ describe(title, () => {
     // If we disable admin approval, review been shown to UI
```

Comment:
Now, if we disable admin approval then reviews are showing immediately.
So, you need to remove approval testcase from the files on which we have this comment and testcase

Reply:
In 2.2.3 - Remove this and add reviewTestCase()
 // If we disable admin approval, review been shown to UI
  // but review status is still pending so approving it via graphql
  approveReviews(user1.name)